### PR TITLE
Add sei live info

### PR DESCRIPTION
**Pull request by Lesigh-3100**
This PR updates the SEI live network details.

### **network.py**
- [x] WSEI address
- [x] RPC endpoint
- [x] Carbon address
- [x] Fastlane Address
- [ ] Stablecoin address
- [ ] Chain Flashloan Tokens
- [x] multicall address
- [ ] BALANCER_VAULT_ADDRESS

### **multichain_addresses.csv** 
- [x] carbon_v1
- [x] dragonswap
- [ ] jellyswap
- [ ] uniswap_v3
- [ ] **Check uniswap V2 fork fees**

### **update static data**
Run terraformer with deployed exchange addresses & validate the following files in the **fastlane_bot/data/blockchain_data/sei/** folder:
- [ ] tokens.csv
- [ ] static_pooldata.csv
- [ ] uniswap_v2_event_mappings.csv
- [ ] uniswap_v3_event_mappings.csv
- [ ] solidly_v2_event_mappings.csv

### uni V3 Router contract check
- [ ] Check router versions for Uni V3 deployments
- [ ] Update routehandler.py with exchanges that use router 01
- [ ] Update _handle_custom_data_extras()

### CODE REVIEW

**Review by barakman — DISMISSED** [review]

File: fastlane_bot/config/network.py [hunk 1]
```diff
@@ -791,21 +791,21 @@ class _ConfigNetworkSei(ConfigNetwork):
     """
 
     NETWORK = S.NETWORK_SEI
-    NETWORK_ID = "1" # TODO
+    NETWORK_ID = "1329" # TODO
     NETWORK_NAME = "sei"
     DEFAULT_PROVIDER = S.PROVIDER_ALCHEMY
-    RPC_ENDPOINT = "https://evm-rpc.arctic-1.seinetwork.io/" # TODO currently Sei devnet
+    RPC_ENDPOINT = "https://evm-rpc.sei-apis.com/?x-apikey="
```
Comment (barakman) [inline fastlane_bot/config/network.py:797]: Probably need to update README.md and env.example as well

**Review by barakman — CHANGES_REQUESTED** [review]

File: fastlane_bot/config/network.py (hunk 1 above)
Comment (zavelevsky) [reply fastlane_bot/config/network.py:797]: @Lesigh-3100 can you please go over the readme and see that it's clear how to use Sei?

File: fastlane_bot/config/network.py [hunk 2]
```diff
@@ -791,21 +791,21 @@ class _ConfigNetworkSei(ConfigNetwork):
     """
 
     NETWORK = S.NETWORK_SEI
-    NETWORK_ID = "1" # TODO
+    NETWORK_ID = "1329" # TODO
```
Comment (zavelevsky) [inline fastlane_bot/config/network.py:794]: @Lesigh-3100 let's handle TODOs

File: fastlane_bot/config/network.py [hunk 3]
```diff
@@ -815,7 +815,7 @@ class _ConfigNetworkSei(ConfigNetwork):
     BALANCER_VAULT_ADDRESS = "0x7ccBebeb88696f9c8b061f1112Bb970158e29cA5" # # TODO Jellyswap on devnet
```
Comment (zavelevsky) [inline fastlane_bot/config/network.py:815]: @Lesigh-3100 let's handle TODOs

File: fastlane_bot/config/network.py (hunk 2 above)
Comment (Lesigh-3100) [reply fastlane_bot/config/network.py]: Done.

File: fastlane_bot/config/network.py (hunk 3 above)
Comment (Lesigh-3100) [reply fastlane_bot/config/network.py]: Done.

File: fastlane_bot/config/network.py (hunk 1 above)
Comment (Lesigh-3100) [reply fastlane_bot/config/network.py:797]: Added the `WEB3_SEI` env example to README. It had already been added to .env.example.

**Review by barakman — APPROVED** [review]

File: main.py [hunk 4]
```diff
@@ -592,7 +592,6 @@ def run(mgr, args, tenderly_uri=None) -> None:
             "b3_two_hop",
             "multi_pairwise_pol",
             "multi_pairwise_all",
```
Comment (zavelevsky) [inline main.py:594]: no need to remove the ','?

**Review by zavelevsky — APPROVED** [review]

**Pull request merged**